src/extension.ts

```diff
@@ -232,7 +232,14 @@ function fetchAndLaunchMetals(context: ExtensionContext, javaHome: string) {
         javaConfig
       );
     },
-    () => {
+    (reason) => {
+      if (reason instanceof Error) {
+        const err = reason as Error;
```

Comment:
No need for casting, if you already used `instanceof` (flow typing should understand that)

Reply:
That's neat, fixed!